spec/features/case_management/tax_return_status_change_spec.rb

```diff
@@ -0,0 +1,33 @@
+require "rails_helper"
+
+RSpec.feature "Change tax return status on a client" do
+  context "As a beta tester" do
+    let(:vita_partner) { create :vita_partner }
+    let(:logged_in_user) { create :user, vita_partner: vita_partner }
```

Comment:
Consider `let(:user)` rather than `let(:logged_in_user)` here, since they're not logged-in yet at the time of the `let`. Non-blocking.

Reply:
changed